scripts/main.py

```diff
@@ -467,6 +466,9 @@ def main():
                 out_path = os.path.join(args.exp_dir, "bigwig_peakfiles")
                 cleanpeakbw = peak2bw(args.cleanpeakfile, args.genome,
                                       out_path)
+                args.cleanbw = gather_files_from_cmdline(
+                    args.cleanbw,
+                    extension=".bw")
 
             # Generate training, validation, holdout intervals files
```

Comment:
Let's change this comment to avoid confusion - we aren't generating training and val files here

Reply:
done